extensions/azurePublishNew/src/api.ts

```diff
@@ -23,3 +24,14 @@ export const getDeployLocations = (accessToken: string, subscription: string): P
     resolve([]);
   });
 };
+
+export const CheckWebAppNameAvailability = async (
+  token: string,
+  webAppName: string,
+  subscriptionId: string
+): Promise<ResourceNameAvailability> => {
+  return {
```

Comment:
what is this doing?

Reply:
This is just a dummy function, will be removed eventually once the API layer is ready.